client/components/ChatMessage.js

```diff
@@ -4,7 +4,14 @@ import { makeStyles } from '@material-ui/core/styles';
 
 const useStyles = makeStyles((theme) => ({
   guessName: {
-    wordBreak: 'break-all',
+    maxWidth: '50%',
+    display: 'inline-block',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+    overflowX: 'hidden',
+    verticalAlign: 'bottom',
```

Comment:
What is the significance of the vertical alignment?

Reply:
For reasons I don't fully understand, `display: inline-block` and `overflow: hidden` together cause the vertical alignment to be off without that. Stolen from here: https://stackoverflow.com/questions/20566710/overflowhidden-displayinline-block-moves-text-upwards